aws_lambda_events/Cargo.toml

```diff
@@ -25,11 +25,11 @@ serde_with = { version = "^1", features = ["json"], optional = true }
 serde_json = "^1"
 bytes = { version = "1", features = ["serde"] }
 chrono = { version = "^0.4.4", default-features = false, features = ["serde", "std"] }
-query_map = { version = "^0.5", features = ["serde"] }
+query_map = { version = "^0.6", features = ["serde"] }
```

Comment:
This one, specifically, bit me in some work I'm doing with [`lambda_http`](https://docs.rs/lambda_http/).